libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -253,7 +258,7 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
                                     current,      // current in centiampere
                                     consumed_mah, // total consumed current in milliampere.hour
                                     consumed_wh,  // consumed energy in hJ (hecto-Joules)
-                                    battery.capacity_remaining_pct(instance),
+                                    percentage,
```

Comment:
is -1 ok here? It was 0

Reply:
This is the intended change. If the autopilot reports it is not measuring the battery capacity (-1), the GCSs can just display battery Voltage instead.